### PR TITLE
Peer program with Christy, Erdem and Andrew

### PR DESCRIPTION
**Pull request by christylaguardia**
#### Single-line Summary
**Today, Christy, Erdem and Andrew paired together. It took about _**

#### Reflect and summarize on your process for each `TODO` item :  
  1. First, we started on installing express and body-parse
  2. Next, we instantiated the app
  3. Then we added the app.use,  app. get and app.listen
  4. Then we updated the form in newform.html and aded the app.post in server.js

#### Checklist (before submitting, fill in each set of square brackets with an 'x')
- [x] We have titled the Pull Request similar to our branch name (ex: 'brian-rick'). 
- [x] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.
- [x] There is no extraneous, unrelated code included in this PR.
- [x] We have summarized our `TODO:` process above.

### CODE REVIEW

**Comment by kbrumer** [comment]
👍

File: starter-code/server.js [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
 // TODO: Include all of the static resources as an argument to app.use()
+app.use(express.static('./public'));
 
-app.get('*', function(request, response) {
-  // TODO: Using the response object, send the index.html file back to the user
-});
+// app.get('*', function(request, response) {
+//   // TODO: Using the response object, send the index.html file back to the user
+//   response.sendFile('index.html', { root: './public' });
+// });
```
Comment (JPGothro) [inline starter-code/server.js]: You didn't complete this TODO, why? The code looks fine...

File: starter-code/server.js [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
+app.get('/newarticle', function(request, response) {
+  response.sendFile('new.html', { root: './public' });
+});
+
+// TODO: (STRETCH) This route will receive a new article from the form page, new.html,
+// and log that form data to the console.
+app.post('/new', urlEncodedParser, function (request, response) {
+  console.log('you made a new article: ', request.body);
+  response.send('you made a new article!');
+});
```
Comment (JPGothro) [inline starter-code/server.js]: Good job above!

File: starter-code/server.js (hunk 1 above)
Comment (kbrumer) [inline starter-code/server.js]: We also noted that index.html was already being served up by line 14. Althoughs TODOs should be TODONE.

**Pull request closed without merging**